packages/app/components/Today/ReleaseNotes.tsx

```diff
@@ -0,0 +1,71 @@
+import { Modal, notification } from "antd";
+import { ReactElement, useState, useEffect } from "react";
+import { useLocalStorage } from "../../hooks/useLocalStorage";
+import { NotionRenderer } from "react-notion";
+
+export default function ReleaseNotes(): ReactElement {
+  const [releaseNotesLastSeen, setReleaseNotesLastSeen] = useLocalStorage(
+    "releaseNotesLastSeen",
+    null
+  );
+  const [releaseNotesLastUpdated, setReleaseNotesLastUpdated] = useState(0);
+  const [notionReleaseNotes, setNotionReleaseNotes] = useState();
+  const [showReleaseNotes, setShowReleaseNotes] = useState(false);
+
+  if (!releaseNotesLastSeen) {
+    setReleaseNotesLastSeen(new Date());
+  }
+
+  useEffect(() => {
+    fetch(
+      "https://notion-api.splitbee.io/v1/page/abba246bfa0847baa2706ab30d0c6c7d"
+    )
+      .then((res) => res.json())
+      .then(
+        (result) => {
+          const timeText =
+            result["beae2a02-249e-4b61-9bfc-81258d93f20d"]?.value?.properties
+              ?.title[0][0];
+          setReleaseNotesLastUpdated(timeText.split("Unix ")[1] * 1000);
+          // Remove the time block and page link block from page
+          result["beae2a02-249e-4b61-9bfc-81258d93f20d"].value.properties.title = []
+          result["4d25f393-e570-4cd5-ad66-b278a0924225"].value.properties.title = []
```

Comment:
This feels a little hacky but I couldn't think of another way to ready and modify the data from the api response. Lmk if there is a better way to do this

Reply:
why do we need a full copy of the notion release notes?

Reply:
We are using Notion as our CMS and the api returns all the data from the page

Reply:
not sure if this will work, but maybe you could do `delete result["beae2a02-249e-4b61-9bfc-81258d93f20d"]` instead?

Reply:
I tried `delete result["beae2a02-249e-4b61-9bfc-81258d93f20d"]` and it was giving me some weird issues. The way it's done seems to work as intended for now